From: Sven Roederer <freifunk@it-solutions.geroedel.de>
Date: Sat, 2 May 2020 01:18:19 +0200
Subject: luci.mk: separate $(LUCI_BUILD_DEPENDS) from generic dependencies

Separate the package specific $(LUCI_BUILD_DEPENDS) from the common dependencies
of LuCI. Moving it closer to the section dealing with the package specific
imports makes it more easy to see on the first view.

Signed-off-by: Sven Roederer <devel-sven@geroedel.de>

diff --git a/luci.mk b/luci.mk
index be11cf99e05ad571cafeff13869b4c87139f4f3f..59433ab8850d19640276331b476628c9b981fe9d 100644
--- a/luci.mk
+++ b/luci.mk
@@ -13,6 +13,8 @@ LUCI_PKGARCH?=$(if $(realpath src/Makefile),,all)
 LUCI_SECTION?=luci
 LUCI_CATEGORY?=LuCI
 
+PKG_BUILD_DEPENDS += $(LUCI_BUILD_DEPENDS)
+
 # Language code titles
 LUCI_LANG.ar=العربية (Arabic)
 LUCI_LANG.bg=български (Bulgarian)
@@ -96,7 +98,7 @@ endef
 PKG_NAME?=$(LUCI_NAME)
 PKG_RELEASE?=1
 PKG_INSTALL:=$(if $(realpath src/Makefile),1)
-PKG_BUILD_DEPENDS += lua/host luci-base/host LUCI_CSSTIDY:csstidy/host LUCI_SRCDIET:luasrcdiet/host $(LUCI_BUILD_DEPENDS)
+PKG_BUILD_DEPENDS += lua/host luci-base/host LUCI_CSSTIDY:csstidy/host LUCI_SRCDIET:luasrcdiet/host
 PKG_CONFIG_DEPENDS += CONFIG_LUCI_SRCDIET CONFIG_LUCI_JSMIN CONFIG_LUCI_CSSTIDY
 
 PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)
